src/plugins/downloader/templates/download-manager.tsx

```
@@ -0,0 +1,379 @@
+  };
+
+  // ─── Render ──────────────────────────────────────────────────────
+
+  return (
+    <>
+      {/* Toggle Button */}
+      <button
+        class={`dm-toggle-btn ${isOpen() ? 'dm-active' : ''}`}
+        onClick={() => setIsOpen(!isOpen())}
+        title="Gestor de Descargas"
+        id="ytmd-download-manager-toggle"
```

ESLint

[error] Props should be sorted alphabetically

(stylistic/jsx-sort-props)